src/main/java/io/quarkus/search/app/quarkusio/QuarkusIO.java

```diff
@@ -275,8 +281,9 @@ private Stream<? extends Guide> translateGuide(Guide guide, Map<Language, Catalo
                         if (!gitInputProvider.isFileAvailable()) {
                             // if  a file is not present we do not want to add such guide. Since if the html is not there
                             // it means that users won't be able to open it on the site, and returning it in the search results make it pointless.
-                            Log.warn("Guide " + guide
-                                    + " is ignored since we were not able to find an HTML content file for it.");
+                            failureCollector.warning(FailureCollector.Stage.TRANSLATION,
+                                    "Guide " + guide
+                                            + " is ignored since we were not able to find an HTML content file for it.");
```

Comment:
Do we need an `INFO` level? Remember that any warning will lead to the github issue being opened, marking `search.quarkus.io` as down on https://status.quarkus.io/...

Reply:
Alternatively, maybe we should not reopen the github issue where there are only warnings. We'd just post a comment.